### PR TITLE
Feature/api key

### PR DESCRIPTION
**Pull request by michalszc**

### CODE REVIEW

File: backend/README.md [hunk 1]
```diff
@@ -9,6 +9,8 @@ SERVER_TIMEOUT_READ=3s
 SERVER_TIMEOUT_WRITE=5s
 SERVER_TIMEOUT_IDLE=5s
 SERVER_DEBUG=true
+SERVER_API_KEY=a9655739cbfed565b6b86dfb4d2a63df
```
Comment (Tosiekdev) [inline backend/README.md:12]: is this secure?

Comment (michalszc) [reply backend/README.md:12]: yap

Comment (Tosiekdev) [reply backend/README.md:12]: git

**Review by Tosiekdev — APPROVED** [review]
pico bello

**Pull request merged**